ginivision/src/main/java/net/gini/android/vision/internal/camera/api/Util.java

```diff
@@ -14,15 +15,45 @@
 public final class Util {
```

Comment:
After reading and presenting Clean Code I should start refactoring code my younger self vomited out. 😁